src/ask/docs/selecting-mode-decision.md

```
@@ -0,0 +1,65 @@
+| `gpt-4o`, `gpt-4-turbo`, `gpt-4.1` | Chat | Standard chat models |
+| `gpt-3.5-turbo` (not -instruct) | Chat | Standard chat model |
+| `gpt-5`, `gpt-5-mini`, `gpt-5-nano` | Chat | Standard chat models |
+| `o1-*`, `o3-*` | Chat | Reasoning models (chat-compatible) |
+| `chatgpt-4o-latest` | Chat | ChatGPT snapshot |
+| `gpt-5-codex`, `gpt-5.2-codex` | Responses | Code execution model, NOT chat-compatible |
+| `gpt-5-pro` | Responses | Specialized model, NOT chat-compatible |
+| `gpt-3.5-turbo-instruct` | Completions | Legacy completion-only model |
+
+## How the Ask Tool Decides
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)